packages/codemods/transforms/v3/radio-card/index.js

```diff
@@ -0,0 +1,63 @@
+const CODEMOD_ANALYSIS = process.env.CODEMOD_ANALYSIS;
+
+module.exports = function ({ source /*, path*/ }, { parse, visit }) {
+  const ast = parse(source);
+
+  return visit(ast, (env) => {
+    let { builders: b } = env.syntax;
+
+    return {
+      ElementNode(node) {
+        if (node.tag === 'Hds::Form::RadioCard') {
+          // filter out the `@layout` attribute
+          const outputAttrs = node.attributes.filter(a => a.name !== '@layout');
+
+          // look up for `@layout`
+          const attr = node.attributes.find(a => a.name === '@layout');
+
+          // update the attributes
+          if (attr && attr.value) {
+            if (!CODEMOD_ANALYSIS) {
+              return [
+                b.element(
+                  { name: node.tag, selfClosing: !node.children.length },
+                  {
+                    attrs: outputAttrs,
+                    children: node.children,
+                    modifiers: node.modifiers,
+                    blockParams: node.blockParams,
+                  }
+                )
+              ];
+            }
+          }
+        } else {
```

Comment:
does this apply to any other kind of node?

Reply:
🤦 I forgot the node name – amended now